ProjFS.Mac/PrjFSKext/public/Message.h

```diff
@@ -29,6 +29,7 @@ typedef enum
     // Responses
     MessageType_Response_Success,
     MessageType_Response_Fail,
+    MessageType_Response_Aborted,
```

Comment:
This is listed in the "responses" section here, and yet it is explicitly disallowed as a response in the message handling code. How important is it to have a new enum value in the first place? I don't think we're doing anything different as a result of this response - do you have future plans for it?

Reply:
Yeah, I was on the fence for this, but ultimately, I think we should be handling this case identically to there not being an active provider for that root in the first place, which I’m not sure is always going to be the same as when an active, responding provider is explicitly making a “failed” response?

Reply:
Yea I can see that. I think I would remove the `_Response_` portion of its name, since it's not a response.

Reply:
Using the same logic, the `receivedResponse` and `response` field names in the struct no longer seemed entirely appropriate, so I've [renamed the "response" portion to "result"](https://github.com/Microsoft/VFSForGit/pull/501/commits/81ed1352b63a2b32e4d17a730a7911d947e7b115) there. Sound sensible?